src/main/java/org/schemaspy/Revision.java

```diff
@@ -24,11 +24,14 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.util.logging.Level;
```

Comment:
Use slf4j

Reply:
Per request, I've changed to slf4j, but there are no other classes using slf4j. There are 28 classes that import `java.util.logging.Logger`